build-tools/xaprepare/xaprepare/ConfigAndData/Configurables.cs

```diff
@@ -353,6 +353,7 @@ public static partial class Paths
 			public static readonly List<string> BundleVersionHashFiles = new List<string> {
 				Path.Combine (BuildPaths.XAPrepareSourceDir, "ConfigAndData", "BuildAndroidPlatforms.cs"),
 				Path.Combine (BuildPaths.XAPrepareSourceDir, "ConfigAndData", "Runtimes.cs"),
+				Path.Combine (BuildPaths.XAPrepareSourceDir, "ConfigAndData", "Runtimes.Code.cs"),
```

Comment:
I'm not sure this is necessary to put this file in the hash files list. The reason I split it out was that I wanted to keep **code** changes from affecting the bundle hash (data that should affect the bundle hash is in `Runtimes.cs`). Code in `Runtimes.Code.cs` is driven by the data in `Runtimes.cs` and while occasionally a change to the former may require a bundle hash change (and we can trigger it by a change to `Runtimes.cs` if need be), mostly edits of `Runtimes.Code.cs` won't be relevant in this regard.

Reply:
Good catch, I made this change before the more recent change to `Runtimes.cs` because without it the bundle wasn't invalidated and the .pdb files for the designer weren't included. Now that the PR contains a change to Runtimes.cs I can revert it.